extensions/markdown-language-features/src/util/links.ts

```diff
@@ -22,13 +22,29 @@ const knownSchemes = [
 ];
 
 export function getUriForLinkWithKnownExternalScheme(link: string): vscode.Uri | undefined {
-	if (knownSchemes.some(knownScheme => isOfScheme(knownScheme, link))) {
+	const cleanLink = stripAngleBrackets(link);
+	if (knownSchemes.some(knownScheme => isOfScheme(knownScheme, cleanLink))) {
 		return vscode.Uri.parse(link);
 	}
 
 	return undefined;
 }
 
+/* Used to strip brackets from the markdown link
+	<http://example.com> will be transformed to
+	http://example.com
+*/
+export function stripAngleBrackets(link: string) {
+	const bracketMatcher = /^<(.*)>$/;
```

Comment:
[nit] Use `.replace` instead to simplify this

Reply:
Absolutely, I agree the match made it complex.

Reply:
This has now been simplified, please see the most recent changes 👍